### DISCUSSION TITLE
Accessing extension path from the extension

### DISCUSSION THREAD

**Discussion by wingyplus** [zed-industries/zed#14952] [Feature Requests]
I wrote the [PowerShell](https://github.com/wingyplus/zed-powershell) extension and try to implement automatic install the language server (PowerShell Editor Services) like other extensions does. The problem is the LSP requires to start through PowerShell, it run on project directory as a working directory, cause LSP to crash because it cannot find the LSP script. 

So my question is how do I access the extension path (for example, `/Library/Application\ Support/Zed` on mac)? Or the zed has any a better way to fix this?

Sorry if I do something wrong, never writing Rust before. 🙏

**Comment by tuugen** [comment]
this would be super helpful to get an update on @zed-intelligence

**Reply by ramaddk** [reply]
did you ever find a proper solution?

in my case is solved it by setting permissions on the dir.

chmod -R 755 "/Users/USERNAME/Library/Application Support/Zed/extensions/work/powershell/powershell-es-v4.4.0/PowerShellEditorServices"

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
